wpiutil/src/main/native/thirdparty/llvm/cpp/llvm/ErrorHandling.cpp

```diff
@@ -84,7 +84,7 @@ void wpi::report_fatal_error(std::string_view Reason, bool GenCrashDiag) {
   }
 
   if (handler) {
-    handler(handlerData, std::string{Reason}, GenCrashDiag);
+    handler(handlerData, Reason.data(), GenCrashDiag);
```

Comment:
If handled takes a const char*, this could pass a non-nul-terminated string to it since string_view is not guaranteed to be nul terminated.

Reply:
So `std::string{Reason}.c_str()` instead?